www/include/monitoring/objectDetails/xml/hostSendCommand.php

```diff
@@ -49,39 +50,55 @@
 }
 $pearDB = new CentreonDB();
 $hostObj = new CentreonHost($pearDB);
-$host_id = $_GET["host_id"];
-$poller = $hostObj->getHostPollerId($host_id);
-$cmd = $_GET["cmd"];
-$sid = session_id();
-$act_type = $_GET["actiontype"];
+
+$hostId = filter_var(
+    $_GET['host_id'] ?? false,
+    FILTER_VALIDATE_INT
+);
+
+$pollerId = $hostObj->getHostPollerId($hostId);
+
+$cmd = $_GET['cmd'];
```

Comment:
$cmd should be an integer. Please add a check

Reply:
`$_GET['cmd'] `is not an integer. For a re-schedule check for example the value of `$_GET['cmd'] `would be "service_schedule_check"

Reply:
ok. Then you need to sanitize it, to avoid a buffer overflow injection or something else -> line 123

Reply:
Need to filter sanitize + CentreonUtils::escapeSecure()